modules/claims_api/spec/requests/metadata_spec.rb

```diff
@@ -85,6 +85,22 @@
             end
           end
         end
+
+        local_bgs_person_services = %i[person]
+        local_bgs_person_methods = %i[find_by_ssn]
+        local_bgs_person_services.each do |local_bgs_person_service|
```

Comment:
Not really blocking but, I assume this is set up this way because `local_bgs_person_services` is expected to grow in the future?  Just curious about the use of a loop and variables with these single values

Reply:
@rockwellwindsor-va Great point! Yes, we will be able to expand on the methods, but not the service. I will remove the loop for the service. TY!